dev/get_version.py

```diff
@@ -1,21 +1,20 @@
-# /// script
-# dependencies = [
-#   "toml",
-# ]
-# ///
 import subprocess
 from pathlib import Path
 
-import toml
+
```

Comment:
do we even need this script at all? `uv version` will do this

Reply:
Great suggestion. Lets use that command.